config/config.go

```diff
@@ -499,6 +507,7 @@ func NewConfig() (c *Config) {
 		IstioLabels: IstioLabels{
 			AppLabelName:       "app",
 			InjectionLabelName: "istio-injection",
+			InjectionLabelRev:  "istio.io/rev",
 			VersionLabelName:   "version",
 		},
 		KialiFeatureFlags: KialiFeatureFlags{
```

Comment:
The IstioUpgradeAction would require a default value.
As an upgrade is not a daily task I'd define it as "false" by default.

Reply:
this is default 'false' in operator

Reply:
now default 'false' here as well

Reply:
Yes, even if it sounds redundant, but there are some explicit defaults in code as well.